src/_data/projects.js

```diff
@@ -0,0 +1,86 @@
+require("dotenv").config();
+const Image = require("@11ty/eleventy-img");
+
+var Airtable = require("airtable");
```

Comment:
not a show stopper, but can we use either `const` or `let` in place of all the `var`s in this file? Can be done after merging